### PR TITLE
New categories menu

### PR DESCRIPTION
**Pull request by antoniotrkdz**
Solved category selection issue on form register students

### CODE REVIEW

**Comment by des-des** [comment]
@antoniotrkdz Duplicate commits on other pulls I have already looked at

**Comment by des-des** [comment]
@antoniotrkdz STOP

**Comment by antoniotrkdz** [comment]
Hi Eoin, I have merged master in and I hope the size of the PR shrinked.
Besides merging master in I made a small change to categories file to correct a mistake, but hopefully this PR should be of an acceptable size.
I am holding to others to be pushed to avoid the the issues we are having now.
Please do not review thw other PR still on list until you merged this.

Sorry again for the inconvenience, I was just tryitng to speed up the thing, but I have ended up slowing it down instead....

File: react-ui/public/react-widgets.css [hunk 1]
```diff
@@ -0,0 +1 @@
...lect-wrapper>.rw-input::-webkit-input-placeholder{color:#999}.rw-state-readonly>.rw-multiselect-wrapper,.rw-state-disabled>.rw-multiselect-wrapper{cursor:not-allowed}.rw-rtl .rw-multiselect-wrapper>.rw-input{float:right}.rw-multiselect-create-tag{border-top:1px #ccc solid;padding-top:5px;margin-top:5px}.rw-multiselect-taglist{margin:0;padding-left:0;list-style:none;display:inline;padding-right:0}.rw-multiselect-taglist>li{display:inline-block;padding-left:5px;padding-right:5px}.rw-multiselect-taglist>li{display:inline-block;margin:1px;padding:.214em .15em .214em .4em;line-height:1.4em;text-align:center;white-space:nowrap;border-radius:3px;border:1px solid #ccc;background-color:#ccc;cursor:pointer}.rw-multiselect-taglist>li.rw-state-focus{background-color:#fff;border:#66afe9 1px solid;color:#333}.rw-multiselect-taglist>li.rw-state-readonly,.rw-multiselect-taglist>li.rw-state-disabled,.rw-multiselect.rw-state-readonly .rw-multiselect-taglist>li,.rw-multiselect.rw-state-disabled .rw-multiselect-taglist>li{cursor:not-allowed;filter:alpha(opacity=65);opacity:.65}.rw-multiselect-taglist>li .rw-btn{outline:0;font-size:115%;line-height:normal}.rw-rtl .rw-multiselect-taglist>li{float:right}
```
Comment (des-des) [inline react-ui/public/react-widgets.css]: You do not need to commit this,

https://jquense.github.io/react-widgets/docs/#/?_k=qj5kx2

you may have to tweek your webpack config to get this working

Comment (antoniotrkdz) [inline react-ui/public/react-widgets.css]: Can be the css from this be include into the primary style.css?

**Comment by antoniotrkdz** [comment]
Hi Eoin,
I have included the react-widget.css in the main style.css, to avoid complicated configurations.
I also mdified the /build directory route to *NOT* require authentication as Maja was't able to access it on heroku after the new auth was merged into master.

**Comment by des-des** [comment]
> I also mdified the /build directory route

@antoniotrkdz why did you do that on this pulls

I have explicitly asked you not to add extra changes to pulls that I have reviewed.

**Pull request merged**